test_scripts/Defects/8_1/2448_Factory_Defaults_Removes_AppInfo_Dat.lua

```diff
@@ -0,0 +1,62 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/2448
+--
+-- Description: Check that SDL deletes application info (resumption data) file during FACTORY_DEFAULTS`
+--
+-- Preconditions:
+-- 1. Core and HMI are started and initialized
+-- 2. Mobile app is registered and activated
+-- 3. HMI sends OnExitAllApplications with reason FACTORY_DEFAULTS
+-- Sequence:
```

Comment:
```suggestion
-- Steps:
```
I propose to use 'Steps' or 'in case' according to the common style with SDL does